docs/UserGuide.md

```diff
@@ -139,6 +140,26 @@ Examples:
 
 - `budget b/500`
 
+### Removing a person who had paid the owes money: `paid`
```

Comment:
Looks good but could you move this up a couple lines so that its in between list and filter such that it follows the same order as the summary list below? Thanks!

Reply:
have reordered, is this what you meant?